website/content/workbooks/workbook-9.md

```diff
@@ -0,0 +1,38 @@
++++
+title="CYF+02 Sprint 3"
+date="01 Jan 2024"    
+versions=["1-1-0"]
+weight=4
++++
+
+# Provisional start date: 6 May 2024
+
+## Study
+
+- [ ] [Distributed Systems: State](../../primers/distributed-software-systems-architecture/state)
+- [ ] [Troubleshooting Primer](../../primers/troubleshooting/)
+- [ ] [Linux Bash I](https://www.bogotobogo.com/Linux/linux_tips2_bash.php) & [Linux Bash II](https://www.bogotobogo.com/Linux/linux_tips2_bash.php) (There are other blog posts in the same series that could be useful to go through in spare time, would recommend running the commands that the blog post suggests for better understanding.)
+- [ ] PHP fastcgi - https://www.php.net/manual/en/install.fpm.php
+      Systemctl - https://www.freedesktop.org/software/systemd/man/systemctl.html and then https://www.redhat.com/sysadmin/linux-systemctl-manage-services
+- [ ] Nginx
+      https://nginx.org/en/docs/
+      https://nginx.org/en/docs/http/request_processing.html
+      https://www.nginx.com/resources/wiki/start/topics/examples/phpfcgi/
+
+## Projects
+
+- [ ] [Multiple servers](../../projects/multiple-servers) Revisit
```

Comment:
Do you remember what the revisiting was for?

Reply:
We used this project to explore Docker, Docker Compose, and GH Actions, as we didn't feel like it warranted a whole project in the end. Basically we said take this project and deploy it with docker and gh, I think?

You can see this here: https://github.com/berkeli/immersive-go/pull/2/commits

There'a an initial implementation and then a month later he comes back and adds Docker.

Reply:
Aha, makes sense :) Yeah, I think this was the project @sre-is-laura was going to co-opt to introduce kubernetes, which sounds like an even better fit than we remembered it to be!